.github/workflows/pytest-daily.yaml

```diff
@@ -71,7 +71,7 @@ jobs:
           set -ex
           export PATH=/composer-python:$PATH
           python -m pip install --upgrade pip wheel
-          python -m pip install -e .[all]
+          python -m pip install --upgrade -e .[all]
```

Comment:
```suggestion
          python -m pip install --upgrade .[all]
```

Reply:
oops, i meant to do that. my fingers just typed something else.